src/hexkit/protocols/dao.py

```diff
@@ -97,6 +97,14 @@ def __init__(self, *, mapping: Mapping[str, str]):
         super().__init__(message)
 
 
+class DbTimeoutError(RuntimeError):
+    """Raised when a database operation timed out."""
+
+    def __init__(self, reason: str):
```

Comment:
Maybe something like `details` or `raw_message` instead of `reason`? Because the reason for the error is the timeout and the reason for the timeout is probably not specified in this message.

Reply:
Changed to `details` here: [89c5d87](https://github.com/ghga-de/hexkit/pull/132/commits/89c5d870b72b8c52bbba90e96a5069c8cb2e877f)